llvm/lib/Passes/PassRegistry.def

```diff
@@ -725,7 +725,6 @@ FUNCTION_PASS_WITH_PARAMS(
       return ExpandFpPass(TM, OL);
     },
     parseExpandFpOptions, "opt-level=N")
-    
```

Comment:
nit: please don't change unrelated part of the code.

Reply:
It was a trailing whitespace, should I keep it?